recipe/meta.yaml

```
@@ -42,17 +45,17 @@ test:
     - kombu.transport.virtual
     - kombu.utils
   source_files:
-    - t/
+    - gh_src
   requires:
+    - hypothesis
+    - botocore
+    - pytest
+    - google-api-core
```

```suggestion
    - hypothesis <7
    - botocore
    - pytest
    - google-api-core
```
https://github.com/celery/kombu/blob/v5.5.4/requirements/test.txt